src/flask_session/sessions.py

```diff
@@ -586,6 +653,18 @@ def __repr__(self):
 
         self.sql_session_model = Session
 
+    def _delete_expired_sessions(self) -> None:
+        try:
+            self.db.session.query(self.sql_session_model).filter(
+                self.sql_session_model.expiry <= datetime.utcnow()
```

Comment:
Both here and in the session creation I would use the database's `now()` function. This would prevent issues due to time synchronization between multiple servers.